ludwig/datasets/goemotions/config.yaml

```diff
@@ -0,0 +1,11 @@
+version: 1.0
+download_urls: 
+  - https://raw.githubusercontent.com/google-research/google-research/master/goemotions/data/train.tsv
+  - https://raw.githubusercontent.com/google-research/google-research/master/goemotions/data/dev.tsv
+  - https://raw.githubusercontent.com/google-research/google-research/master/goemotions/data/test.tsv
+download_filenames:
```

Comment:
maybe this can be renames `split_filenames` ?

Reply:
great idea!